sr_port/bm_getfree.c

```diff
@@ -80,7 +80,7 @@ block_id bm_getfree(block_id hint, boolean_t *blk_used, unsigned int cw_work, cw
 	sm_uc_ptr_t	bmp;
 	block_id	bml, hint_cycled, hint_limit;
 	block_id_ptr_t	b_ptr;
-	int		cw_set_top, depth = 0, lcnt;
+	int		cw_set_top, depth, lcnt;
```

Comment:
The commit message says the following. Now that a new scan has been done, can we update the below to reflect only those that did get fixed.

- sr_port/bm_getfree.c - Issues: free_bit (line 216), map_size (line 216), cs1 (line 223),
                         and depth (line 221) are all uninitialized. Additionally,
			 blkhist.cr and blkhist.cycle undefined at line 238.
  - Loop changed so always do one iteration. Will do a new scan and check to see which of
    these issues are fixed by the loop change and which remain.

Reply:
Yes. I'll update the commit comment. Note that bm_getfree.c does not show up in the scan issues list anymore.

Reply:
Nice. Like I had hoped based on the for -> do/while suggestion.